### PR TITLE
Feature/tri 759 testbranch for external functions

### PR DESCRIPTION
**Pull request by ds-alexander-bulgakov**
Complete tavern-tests for TRI-759 with all required tests and function. Several changes in POSTMAN-Collections included.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -58,3 +58,4 @@ dependency-reduced-pom.xml
 ### Local files ###
 dev/local/*
 /api-tests/variable.env
+api-tests/__pycache__/tavern_helpers.cpython-310.pyc
```
Comment (ds-jhartmann) [inline .gitignore:61]: I think its better to just ignore the whole `__pychache__` folter like this: `api-tests/__pycache__/*`

File: api-tests/testikus.tavern.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+test_name: Test for testing
```
Comment (ds-jhartmann) [inline api-tests/testikus.tavern.yaml:1]: Isn't this just a file where you tried out different things? In this case it's probably not necessary to commit it

File: .gitignore (hunk 1 above)
Comment (ds-jhartmann) [reply .gitignore:61]: There is also a conflict with .gitignore on main which need to be resolved

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: api-tests/testikus.tavern.yaml (hunk 2 above)
Comment (ds-alexander-bulgakov) [reply api-tests/testikus.tavern.yaml]: yes, deleted.

**Review by ds-jhartmann — APPROVED** [review]

**Pull request merged**